### ISSUE TITLE
[a11y] seizures triggering flash detector

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#170712]
### Use case

https://www.w3.org/WAI/WCAG21/Understanding/three-flashes-or-below-threshold
https://www.w3.org/WAI/WCAG21/Understanding/three-flashes

A flash definition is change in luminance of 20 cd/m2 or more where the darker image is below 160 cd/m2. The WCAG minimum requirement (level A) says a flash that has more than 3 times per-second with area larger or equal to 341 x 256 out of a  1024 x 768 screen are considered violation.

The level AAA says flash of any size more than 3 times per-second is violation.

Flutter should have the ability to read screen pixel to detect whether a given animation will violate the success criteria

### Proposal

Have either a runtime check on animation/renderobject/layer tree or checks in unittest to detect whether the app can trigger seizures
